index.html

```diff
@@ -2956,10 +2956,10 @@ <h3><dfn>Writing or pushing content</dfn></h3>
 
   <section> <h3>Listening for content</h3>
     <p>
-      In order to receive <a>NFC content</a>, the client needs to attach an
-      event listener for the "`reading`" event on an
-      {{NFCReader}} instance and then activate it by calling
-      <a>NFCReader.scan()</a>.
+      To listen for <a>NFC content</a>, the client MUST activate an
+      {{NFCReader}} instance by calling <a>NFCReader.start()</a>. When attaching
+      an event listener for the "`reading`" event on it, <a>NFC content</a> is
```

Comment:
start() will or has become scan()?

Reply:
`scan`: https://w3c.github.io/web-nfc/#the-scan-method

Reply:
Confused since this text uses `NFCReader.start()`, shouldn't that be `NFCReader.scan()`?

Reply:
I was confused. Sorry for this.
I've sent https://github.com/w3c/web-nfc/pull/306